src/coreclr/jit/codegencommon.cpp

```diff
@@ -5534,22 +5953,232 @@ void CodeGen::genPopCalleeSavedRegisters(bool jmpEpilog)
     noway_assert(compiler->compCalleeRegsPushed == popCount);
 }
 
-#endif // TARGET*
-
-// We need a register with value zero. Zero the initReg, if necessary, and set *pInitRegZeroed if so.
-// Return the register to use. On ARM64, we never touch the initReg, and always just return REG_ZR.
-regNumber CodeGen::genGetZeroReg(regNumber initReg, bool* pInitRegZeroed)
+#elif defined(TARGET_LOONGARCH64)
+void CodeGen::genPopCalleeSavedRegisters(bool jmpEpilog)
```

Comment:
Please put this in codegenloongarch64.cpp.

I'm going to move the other ones to their respective architecture-specific files.

Reply:
OK, Thanks.